GPU/Common/VertexDecoderCommon.h

```diff
@@ -300,7 +300,7 @@ class VertexReader {
 	bool hasNormal() const { return decFmt_.nrmfmt != 0; }
 	bool hasUV() const { return decFmt_.uvfmt != 0; }
 	bool isThrough() const { return (vtype_ & GE_VTYPE_THROUGH) != 0; }
-	bool skinningEnabled() const { return vertTypeIsSkinningEnabled(vtype_); }
+	bool skinningEnabled() const { return false; }
```

Comment:
Well, these can be removed (`numBoneWeights()` too), which I did in the half-step pull.  Software uses them.

-[Unknown]